packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -99,27 +99,28 @@ - (void)setZIndex:(int)zIndex {
     self.marker.zIndex = zIndex;
 }
 
-- (UIImage *)addClusterMarkerText:(NSString *)string {
+
+- (UIImage *)clusterMarkerImageWithText:(NSString *)string {
     CGSize size = [_iconImage size];
-    UIFont *textFont = [UIFont fontWithName:[self path] size:24];
+    UIFont *textFont = [UIFont fontWithName:self.fontPath size:24];
```

Comment:
Since this initializer is optional, I think it'd be a good idea to add a default font here, WDYT? 
https://developer.apple.com/documentation/uikit/uifont/1619041-init

Reply:
Good idea, but would there be any case a default font might be used?

Reply:
Maybe if the system failed to load the fonts? I don't really know what problem can happen, but they made the initializer failable for a reason hehe